dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -16,6 +16,7 @@
 
 package io.dockstore.client.cli.nested;
 
+import com.google.api.client.repackaged.com.google.common.base.Strings;
```

Comment:
Ah, you shouldn't import from the repackaged class. Try to import from Guava directly. That way, we should not need the extra dependency above.